ext/standard/basic_functions.stub.php

```diff
@@ -198,3 +198,11 @@ function htmlentities(string $string, int $quote_style = ENT_COMPAT, ?string $en
 function get_html_translation_table(int $table = HTML_SPECIALCHARS, int $quote_style = ENT_COMPAT, string $encoding = "UTF-8"): array {}
 
 /* }}} */
+
+/* assert.c */
+
+/** @params string|bool $assertion */
```

Comment:
Hm... this is not quite right, in that the string case has been dropped in PHP 8. Now the assertion is always treated as a boolean. But it's not required to be boolean, it's just converted to one. So I think the best type here might just be `mixed`.